WordPress/Classes/Networking/ReaderPostServiceRemote.m

```diff
@@ -287,6 +307,13 @@ - (RemoteReaderPost *)formatPostDictionary:(NSDictionary *)dict
     post.wordCount = [dict numberForKey:PostRESTKeyWordCount];
     post.readingTime = [self readingTimeForWordCount:post.wordCount];
 
+    NSDictionary *railcar = [dict dictionaryForKey:PostRESTKeyRailcar];
+    if (railcar) {
+        NSError *error;
+        NSData *railcarData = [NSJSONSerialization dataWithJSONObject:railcar options:NSJSONWritingPrettyPrinted error:&error];
```

Comment:
WTF


On August 15, 2016 at 5:30:31 PM, Brent Coursey (notifications@github.com)
wrote:

In WordPress/Classes/Networking/ReaderPostServiceRemote.m
<https://github.com/wordpress-mobile/WordPress-iOS/pull/5866#discussion_r74839078>
:


> @@ -287,6 +307,13 @@ - (RemoteReaderPost *)formatPostDictionary:(NSDictionary *)dict
>      post.wordCount = [dict numberForKey:PostRESTKeyWordCount];
>      post.readingTime = [self readingTimeForWordCount:post.wordCount];
>
> +    NSDictionary *railcar = [dict dictionaryForKey:PostRESTKeyRailcar];
> +    if (railcar) {
> +        NSError *error;
> +        NSData *railcarData = [NSJSONSerialization dataWithJSONObject:railcar options:NSJSONWritingPrettyPrinted error:&error];

There may be a reason for it, but do we want to use
NSJSONWritingPrettyPrinted here?

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/wordpress-mobile/WordPress-iOS/pull/5866/files/c050d8a5d6ec98bddc2a80704233507bf18b95df#r74839078>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/AAqNuAXodIBQGolLXj9Owd5QJ61zNsHmks5qgNp2gaJpZM4JksWI>
.